services/movie_session.py

```diff
@@ -42,3 +41,9 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling the `DoesNotExist` exception for `MovieSession.objects.get(id=session_id).delete()` to avoid unhandled exceptions if the session ID is not found.